inc/providers/class-core-sitemaps-users.php

```diff
@@ -98,14 +98,28 @@ public function get_public_post_authors_query( $page_num = 1 ) {
 		// We're not supporting sitemaps for author pages for attachments.
 		unset( $public_post_types['attachment'] );
 
-		$query = new WP_User_Query(
+		/**
+		 * Filters the query arguments for authors with public posts.
+		 *
+		 * Allows modification of the authors query arguments before querying.
+		 *
+		 * @see WP_User_Query for a full list of arguments
+		 *
+		 * @since 5.5.0
+		 *
+		 * @param array $args An array of WP_User_Query arguments.
+		 */
+		$args = apply_filters(
+			'core_sitemaps_user_query_args',
```

Comment:
Following up on the previous comment, this filter is much nicer to use than the ones currently in place for posts and taxonomies. You only need to modify this single filter to affect both the users URL list _and_ the users "max num pages" count. Let's apply the same idea for posts and taxonomies.

Only nit-pick here, this should also use plural "users" like elsewhere, i.e. `core_sitemaps_users_query_args`.